scripts/perf/deploy_horicromtal/deploy_horicromtal.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -euo pipefail
+
+source scripts/perf/helper.inc.sh
+
+DB_PASS=$(read_path_from_vault_json "secret/dsp/cromwell/perf" '.data.db_pass')
+
+read_service_account_from_vault
+	
+function join() { local IFS=","; echo "$*"; }
+
+runner_metadata=(
+  "BUILD_NUMBER=$BUILD_NUMBER"
+  "BUILD_TAG=$BUILD_TAG"
+  "CLEAN_UP=false"
+  "CLOUD_SQL_INSTANCE=$CLOUD_SQL_INSTANCE"
+  "CROMWELL_DB_USER=cromwell"
+  "CROMWELL_DB_PASS=$DB_PASS"
+  "CROMWELL_DOCKER_IMAGE=$CROMWELL_DOCKER_IMAGE"
+  "CROMWELL_PROJECT=broad-dsde-cromwell-perf"
+  "CROMWELL_BUCKET=$GCS_BUCKET"
+  "CROMWELL_STATSD_HOST=10.128.0.4"
+  "CROMWELL_STATSD_PORT=8125"
+  "CROMWELL_PERF_SCRIPTS_BRANCH=${REPO_BRANCH}"
+  "GCS_REPORT_PATH=${GCS_REPORT_PATH}"
+  "GCS_REPORT_BUCKET=${GCS_REPORT_BUCKET}"
+  "TEST_CASE_DIRECTORY=${TEST_CASE_DIRECTORY}"
+)
+
+reader_metadata=(
+  "BUILD_NUMBER=$BUILD_NUMBER"
+  "BUILD_TAG=$BUILD_TAG"
+  "CLEAN_UP=false"
+  "CLOUD_SQL_INSTANCE=$CLOUD_SQL_INSTANCE"
+  "CROMWELL_DB_USER=cromwell"
+  "CROMWELL_DB_PASS=$DB_PASS"
+  "CROMWELL_DOCKER_IMAGE=$CROMWELL_DOCKER_IMAGE"
+  "CROMWELL_PROJECT=broad-dsde-cromwell-perf"
+  "CROMWELL_BUCKET=$GCS_BUCKET"
+  "CROMWELL_STATSD_HOST=10.128.0.4"
```

Comment:
Is this the one we set up ourselves? I should check to make sure it's still in good shape.

Incidentally, if it is the one we set up, the answer to your question about URL in the Google Doc is `broad.io/batch-grafana`

Reply:
Yes it is, based on the IP `10.128.0.4`, and I have found it is out of disk and needing attention

Reply:
Instance is now rehabilitated with a 10x larger disk